### ISSUE TITLE
Dead link to showdown-katex

### ISSUE THREAD

**Issue by richb-hanover** [obedm503/bootmark#31]
I'm looking to migrate from strapdown.js so that I can put my markdown text inside a `<div>`

Bootmark seems to check all the boxes (I'm already using jQuery, it can read a  separate Markdown file with data-src, documentation is good, etc...)

BUT - there are no examples. There's a purported one, but it gives a 404: https://obedm503.github.io/katex-latex/

Am I missing a simple example showing how to drop it in? Many thanks!

**Comment by richb-hanover** [comment]
**Update:** Further rummaging around in `/docs` brought me to https://github.com/obedm503/bootmark/blob/master/docs/markdown-cheatsheet.html, which offered the following snippet that got me on the air. 

```
<script src="https://unpkg.com/bootmark@latest/dist/bootmark.bundle.min.js"></script>
    <script>
      $.fn.bootmark.options.html = {
        theme: 'journal',
        toc: false
      };
```
Nevertheless, it would be good to fix the doc's... Thanks again!

**Comment by obedm503** [comment]
https://obedm503.github.io/bootmark/docs/examples.html

each section has an "examples" section with examples. you can  "view source" on all the examples to see how they work. or see the direct source code here: https://github.com/obedm503/bootmark/tree/master/docs

**Comment by richb-hanover** [comment]
Yes, I now see that directory has all kinds of good examples. But there's still a problem. (And I must  apologize for my initial crappy bug report. Please allow me to do better this time.) 

**The Problem:** On the main [Bootmark Examples](https://obedm503.github.io/bootmark/docs/examples.html) page at https://obedm503.github.io/bootmark/docs/examples.html, the second paragraph says:

> you can now use an unlimited amount of elements and define global options for all to use. See [this for an example.](https://obedm503.github.io/katex-latex/)

The text "this for an example" links to [katex-latex](https://obedm503.github.io/katex-latex/), which gives a 404...

[ And thanks for bootmark - it worked exactly as  I hoped. ]

**Comment by obedm503** [comment]
This this is a dead link report. The correct link should be https://github.com/obedm503/showdown-katex
